ta4j-core/src/main/java/org/ta4j/core/Decimal.java

```diff
@@ -175,6 +175,19 @@ public Decimal pow(int n) {
         }
         return new Decimal(delegate.pow(n, MATH_CONTEXT));
     }
+	
+    /**
+     * Returns a {@code Decimal} whose value is <tt>(this<sup>n</sup>)</tt>.
+     * @param n power to raise this {@code Decimal} to.
+     * @return <tt>this<sup>n</sup></tt>
+     * @see BigDecimal#pow(int, java.math.MathContext)
+     */
+    public Decimal exp(int n) {
+        if (this == NaN) {
+            return NaN;
+        }
+        return Decimal.valueOf(Math.E).pow(n);
```

Comment:
`Returns a {@code Decimal} whose value is <tt>(this<sup>n</sup>)</tt>. ` exactly what the javadoc of already existing `Decimal.pow(int n)` does

`Decimal.valueOf(Math.E).pow(n); ` why?

Reply:
Please feel free to delete, I played with my repo and forgot to be within z-score-branch..The strange thing because of the https://github.com/ta4j/ta4j/issues/103 with the RecursiveCachedIndicator which gives me every time a stackoverflow error so slow calculations. I made something like this Double.valueOf(Math.exp()).. the need for conversion. So how can I prevent this "stackoverflow error" with my Indicators??